### ISSUE TITLE
None activation for UNet output.

### ISSUE THREAD

**Issue by ternaus** [qubvel/segmentation_models.pytorch#9]
It looks like it is sigmoid or softmax applied to the output layer of the UNet architecture.

It would be nice to have an option not to apply anything.

The workaround is:

```
 def activation(x): x

model = smp.Unet('resnet34', encoder_weights='imagenet', activation=activation)

```

But it would be nice to do it without this hack.
